assets/src/components/p4_button/edit.js

```diff
@@ -15,107 +16,71 @@ import classnames from 'classnames';
  */
 import { __ } from '@wordpress/i18n';
 import { useCallback, useState } from '@wordpress/element';
-import { compose } from '@wordpress/compose';
 import {
+	Button,
+	ButtonGroup,
 	KeyboardShortcuts,
 	PanelBody,
 	RangeControl,
 	TextControl,
 	ToggleControl,
-	withFallbackStyles,
 	ToolbarButton,
 	ToolbarGroup,
 	Popover,
 } from '@wordpress/components';
 import {
 	BlockControls,
-	__experimentalUseGradient,
-	ContrastChecker,
 	InspectorControls,
-	__experimentalPanelColorGradientSettings as PanelColorGradientSettings,
 	RichText,
-	withColors,
+	useBlockProps,
 	__experimentalLinkControl as LinkControl,
+	__experimentalUseEditorFeature as useEditorFeature,
 } from '@wordpress/block-editor';
 import { rawShortcut, displayShortcut } from '@wordpress/keycodes';
-import { link } from '@wordpress/icons';
+import { link, linkOff } from '@wordpress/icons';
```

Comment:
These icons are also unused (in fact I don't think we even include `wordpress/icons` 🤔)